application/build.gradle.kts

```diff
@@ -9,7 +9,7 @@ buildscript {
 }
 
 // Enjoy easiest way to configure your Android project
-androidProjectConfiguration(
+androidProjectDefaultConfiguration(
```

Comment:
I would prefer to keep `androidProjectConfiguration` and name extra configuration with name like 
`androidPojectConfigurationExtension` or `android ProjectConfigurationOverride`

the reason behind it - user experience, more complex things should require more effort(longer function name)